conduct.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Code of conduct
+layout: default
+---
+
+# Code of conduct
+
+---
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
```

Comment:
> It is when this is about being harassment-free.


:+1: as a little offtopic, this is an issue I have with freenode's new policy.

* https://www.facebook.com/mikaelahmsuomalainen/posts/972290142865799